### PR TITLE
Add reboot and shutdown service to synology_dsm

### PR DESCRIPTION
**Pull request by mib1185**
## Proposed change

This PR make use of new features in [python-synology lib v1.0.0](https://github.com/ProtoThis/python-synology/releases/tag/1.0.0) and add services to reboot and shutdown a DSM.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#42697
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR.
Please be sure to start with a clean branch when opening a PR for a new feature. You can git rebase this PR/branch to clean it up.

**Comment by mib1185** [comment]
> There are unrelated commits in this PR.
> Please be sure to start with a clean branch when opening a PR for a new feature. You can git rebase this PR/branch to clean it up.

Hi @frenck should I branch from `current` or from `next` ?

**Comment by klaasnicolaas** [comment]
I see changes that you also apply in this pull request: https://github.com/home-assistant/home-assistant.io/pull/15413/files

For new features / integrations you create a branch that descends from the `next` branch. Because your current `next` branch is related to your other pull request and you made a copy of it (`synology-dsm_add-services`) for this pull request, your commits have been mixed up.

**Comment by mib1185** [comment]
@klaasnicolaas thanks for explanation. I will rebase as soon #15413 is merged.

File: source/_integrations/synology_dsm.markdown [hunk 1]
```diff
@@ -159,3 +159,17 @@ For each disk:
 
 ## Cameras
 - `{camera_name}`: Displays cameras added in [Surveillance Station](https://www.synology.com/en-us/surveillance).
+
+## Services
+
+- `synology_dsm.reboot`: Reboot the by `serial` specified DSM. If only one DSM is configured, `serial` is not requiered.
```
Comment (frenck) [inline source/_integrations/synology_dsm.markdown]:
```suggestion
### Service `synology_dsm.reboot`

Reboot the by `serial` specified DSM. If only one DSM is configured, `serial` is not required.
```

File: source/_integrations/synology_dsm.markdown [hunk 2]
```diff
@@ -159,3 +159,17 @@ For each disk:
...
+
+  | Service data attribute | Required | Description |
+  | ---------------------- | -------- | ----------- |
+  | `serial` | yes, when multiple NAS are configured | serial of DSM |
+
+- `synology_dsm.shutdown`: Shutdown the by `serial` specified DSM. If only one DSM is configured, `serial` is not requiered.
```
Comment (frenck) [inline source/_integrations/synology_dsm.markdown]:
```suggestion
### Service `synology_dsm.shutdown`

Shutdown the by `serial` specified DSM. If only one DSM is configured, `serial` is not required.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/synology_dsm.markdown [hunk 3]
```diff
@@ -159,3 +159,19 @@ For each disk:
 
 ## Cameras
 - `{camera_name}`: Displays cameras added in [Surveillance Station](https://www.synology.com/en-us/surveillance).
+
+## Services `synology_dsm.reboot`
```
Comment (frenck) [inline source/_integrations/synology_dsm.markdown]:
```suggestion
## Services

### Service `synology_dsm.reboot`
```

File: source/_integrations/synology_dsm.markdown [hunk 4]
```diff
@@ -159,3 +159,19 @@ For each disk:
...
+
+Reboot the by `serial` specified DSM. If only one DSM is configured, `serial` is not requiered.
+
+  | Service data attribute | Required | Description |
+  | ---------------------- | -------- | ----------- |
+  | `serial` | yes, when multiple NAS are configured | serial of DSM |
+
+## Services `synology_dsm.shutdown`
```
Comment (frenck) [inline source/_integrations/synology_dsm.markdown]:
```suggestion
### Service `synology_dsm.shutdown`
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/synology_dsm.markdown [hunk 5]
```diff
@@ -159,3 +159,21 @@ For each disk:
 
 ## Cameras
 - `{camera_name}`: Displays cameras added in [Surveillance Station](https://www.synology.com/en-us/surveillance).
+
+## Services
+
+### Service `synology_dsm.reboot`
+
+Reboot the by `serial` specified DSM. If only one DSM is configured, `serial` is not requiered.
```
Comment (Quentame) [inline source/_integrations/synology_dsm.markdown:167]:
```suggestion
Reboot the specified NAS by `serial`. If only one DSM is configured, `serial` is optional.
```

File: source/_integrations/synology_dsm.markdown [hunk 6]
```diff
@@ -159,3 +159,21 @@ For each disk:
...
+
+  | Service data attribute | Required | Description |
+  | ---------------------- | -------- | ----------- |
+  | `serial` | yes, when multiple NAS are configured | serial of DSM |
+
+### Service `synology_dsm.shutdown`
+
+Shutdown the by `serial` specified DSM. If only one DSM is configured, `serial` is not requiered.
```
Comment (Quentame) [inline source/_integrations/synology_dsm.markdown:175]:
```suggestion
Shutdown the specified NAS by `serial`. If only one DSM is configured, `serial` is optional.
```

**Review by Quentame — CHANGES_REQUESTED** [review]
requiered --> required

**Comment by Quentame** [comment]
Adding a fix

**Comment by Quentame** [comment]
Fix typo PR #15615

**Pull request merged**